packages/components/src/combobox-control/index.tsx

```diff
@@ -396,6 +399,7 @@ function ComboboxControl( props: ComboboxControlProps ) {
 							__experimentalRenderItem={
 								__experimentalRenderItem
 							}
+							isLoading={ isLoading }
```

Comment:
Instead of passing the `isLoading` prop to `SuggestionsList`, could instead use the `isLoading` boolean flag to render the component conditionally (something like `isExpanded && ! isLoading`) ?

That would allow us to focus the changes only on the `ComboboxControl` component.

Reply:
Ah that makes sense - just hide the suggestion list entirely when the suggestions are still loading. I will try that.

Reply:
try in https://github.com/WordPress/gutenberg/pull/68927/commits/3a357c49486459f4c74e7911396ab08c174fd323